### PR TITLE
Update navigation

### PR DESCRIPTION
**Pull request by thisandagain**
Blocked by GH-1261
Resolves GH-1262

image

### CODE REVIEW

**Comment by mewtaylor** [comment]
@chrisgarrity is this unblocked now?

**Comment by chrisgarrity** [comment]
As soon as I merge https://github.com/LLK/scratch-www/pull/1357 it should be unblocked. I just want to recheck the build-locale after changing the script that generates the localized-urls.json file. It's currently building, so very, very soon I hope.

**Comment by chrisgarrity** [comment]
@mewtaylor Unblocked!

File: src/components/navigation/www/navigation.jsx [hunk 1]
```diff
@@ -202,21 +202,16 @@ var Navigation = React.createClass({
                             <FormattedMessage id="general.explore" />
                         </a>
                     </li>
-                    <li className="link discuss">
-                        <a href="/discuss">
-                            <FormattedMessage id="general.discuss" />
```
Comment (chrisgarrity) [inline src/components/navigation/www/navigation.jsx]: If this is the only use of general.discuss, perhaps we should also remove it from the l10n.json file.

**Comment by mewtaylor** [comment]
On Mac OSX 10.12, Chrome 58:
* [x] - On homepage logged out, I see nav as "Create, Explore, Tips, About"
* [x] - On homepage logged in, I see nav as "Create, Explore, Tips, About"
* [x] - On mystuff, I see nav as "Create, Explore, Tips, About"
* [x] - On `/discuss` logged out, I see nav as "Create, Explore, Tips, About"
* [x] - On my profile logged in, I see nav as "Create, Explore, Tips, About"
* [x] - On on tips page logged in, I see nav as "Create, Explore, Tips, About"

**Pull request merged**